src/simtools/runners/corsika_simtel_runner.py

```diff
@@ -42,26 +43,37 @@ def __init__(
         sim_telarray_seeds=None,
     ):
         self._logger = logging.getLogger(__name__)
+        if not isinstance(corsika_config, list):
+            corsika_config = [corsika_config]
         self.corsika_config = corsika_config
+        # the main corsika config is the one used to define the CORSIKA specific parameters.
+        # The others are used for the array configurations. The name "main" is a bit misleading.
+        self.main_corsika_config = corsika_config[0]
```

Comment:
I suggest to call it `base_corsika_config`. This indicates that it might be overwritten.

Reply:
Done, though not because I think it will actually be overwritten, but rather because "base" is a better name.